tests/virtual_machines_test.go

```diff
@@ -403,6 +403,12 @@ var _ = Describe("Virtual Machines", func() {
 		})
 		//2633
 		Context("When we re-apply a failed VM yaml", func() {
+			totalTimeout := time.Duration(0)
+			BeforeAll(func() {
+				vmFailCleanupWaitTime, err := getVmFailCleanupWaitTime()
+				Expect(err).ToNot(HaveOccurred(), "Should successfully get wait-time argument")
+				totalTimeout = timeout + vmFailCleanupWaitTime
```

Comment:
Add a comment here explaining that we add some time window apart form vmFailCleanup to be sure that we hit it.

Reply:
Can we reword this comment so it will be clear why we need wait-time 
> It is necessary to extend the timeout given to VM creation,
> by the time we wait until a mac is released from vmWait-ConfigMap